### PR TITLE
data : akshaykamath45

### PR DESCRIPTION
**Pull request by akshaykamath45**

### CODE REVIEW

**Review by Sindhuinti — CHANGES_REQUESTED** [review]
Please check the json format here https://linkfree.eddiehub.io/docs/quickstart

**Comment by akshaykamath45** [comment]
@Sindhuinti All checks have been passed,Is this correct,Would it get merged?

File: data/akshaykamath45.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/akshaykamath_45",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/akshaykamath45",
+      "icon": "FaGitHub"
```
Comment (Sindhuinti) [inline data/akshaykamath45.json:21]: Also change FaGitHub to `FaGithub`

**Review by Sindhuinti — CHANGES_REQUESTED** [review]

File: data/akshaykamath45.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  "name": "Akshay Kamath",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Learning  **Web Development**.Computer Engineering Student",
+  "avatar":"https://github.com/akshaykamath45.png",
+  "links": [
+     {
+      "name": "LinkedIn",
+      "url": "https://www.linkedin.com/in/akshaykamath45/",
+      "icon": "FaLinkedIn"
```
Comment (Sindhuinti) [inline data/akshaykamath45.json:11]: Please change FaLinkedIn to `FaLinkedin`

**Review by Sindhuinti — CHANGES_REQUESTED** [review]

File: data/akshaykamath45.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Akshay Kamath",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Learning  **Web Development**.Computer Engineering Student",
+  "avatar":"https://github.com/akshaykamath45.png",
```
Comment (Pradumnasaraf) [inline data/akshaykamath45.json:6]:
```suggestion
```

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
Hey,

In the recent update, we don't need to add `avatar:` anymore, it will auto-pick your GitHub profile picture. So, please remove that.

Also, do the above suggestions

Updated docs: https://linkfree.eddiehub.io/docs/quickstart

File: data/akshaykamath45.json (hunk 3 above)
Comment (akshaykamath45) [reply data/akshaykamath45.json]: I have directly used  `Commit suggestion`,is it fine

**Comment by kumarsonsoff3** [comment]
Closing this one being Duplicate of #4229. The latest one which was good to go has already been merged

**Pull request closed without merging**